### ISSUE TITLE
Fix GitHub Action Issue Trigger 

### ISSUE THREAD

**Issue by kcoronel** [hackforla/CivicTechJobs#579]
### Overview
When creating a new issue team members are getting a notification of an issue trigger. Relevant links are listed under Resources.
Who ever takes on this issue should write out the Action Items needed to solve the issue.  

### Resources/Instructions
- https://github.com/orgs/hackforla/projects/37?pane=issue&itemId=67501179
- https://github.com/hackforla/CivicTechJobs/actions/runs/11020112231

**Comment by LoTerence** [comment]
### Action Items
- [ ] Need someone to investigate what is happening here, what is this action doing, what is it supposed to do, why is it bugging out now.
- [ ] Once we know what its supposed to do, we can decide if we want to keep and fix it if its an easy fix, or just remove it and turn it off altogether

**Comment by LoTerence** [comment]
screenshot:
![image](https://github.com/user-attachments/assets/538937ab-9147-4bb3-94b8-79dbfec27f33)

**Comment by bzzz-coding** [comment]
I examined the GitHub action error and YAML file and now understand that:

- The action was supposed to put newly opened issues in the "new issue approval" column on the project board.

- The reason it hasn't been working is because we have a new project board, so either the token or the IDs used in the code (or both) are obsolete and GitHub couldn't find the project board.

- The new project board has some handy settings for workflow controls now, so I played around with it and set up a workflow that does exactly what the action was trying to do.

- I tested the workflow by opening new issues, and it indeed worked, so I think we can delete this GitHub action.

@LoTerence if this sounds good to you, I can submit a pull request to the develop branch. However, to actually stop this GitHub action (and its error), the change will need to be merged into the main branch eventually, correct?
